package.json

```diff
@@ -26,12 +21,57 @@
     "@types/jest": "^27.4.0",
     "@types/node": "^16.11.12",
     "jest": "^27.4.7",
+    "jest-junit": "^13.2.0",
+    "rimraf": "^3.0.2",
     "ts-jest": "27.1.2",
     "typescript": "^3.9.10"
   },
+  "jest": {
+    "preset": "ts-jest",
+    "testEnvironment": "node",
+    "collectCoverage": true,
+    "coverageDirectory": "dist/tests/coverage",
+    "coverageProvider": "v8",
+    "coverageReporters": [
+      "text",
+      "cobertura"
+    ],
+    "reporters": [
+      "default",
+      [
+        "jest-junit",
+        {
+          "outputDirectory": "dist/tests",
+          "outputName": "report.xml"
+        }
+      ]
+    ]
+  },
   "scripts": {
+    "clean": "rimraf dist",
+    "clean-all": "rimraf dist package-lock.json awslambda-auto.js ./**/*.d.ts ./**/*.js.map tests/unit/*.js src/**/*.js && rm -rf node_modules",
+    "clean-cache": "npm cache clean --force",
+    "prebuild": "npm i",
     "build": "tsc",
-    "test": "jest"
+    "rebuild": "npm run clean && npm run buid",
+    "pretest": "npm i",
+    "test": "jest --colors",
+    "prepack": "npm run build",
+    "postpack": "npm run init-layer && npm run move-tarball && npm run update-layer-version && npm run update-layer-dependency && npm run optimize-layer && npm run zip-up-layer && npm run move-zip",
```

Comment:
I think we should introduce some coarse grained steps that would also help reduce the length of this command. For e.g., `init-layer` could include `move-tarball`. We could have an `update-layer` that combines `update-layer-version` and `update-layer-dependency`. We could combine `optimize-layer` and `zip-up-layer` into a single command

Reply:
Added this [commit](https://github.com/asserts/asserts-aws-lambda-layer-js/pull/26/commits/4444a389d8f572878188a07ba55e2be209223b1c)
Let me know if this works for you?